src/Compilers/Core/Portable/Emit/EditAndContinue/DeltaMetadataWriter.cs

```diff
@@ -633,6 +607,47 @@ protected override void CreateIndicesForNonTypeMembers(ITypeDefinition typeDef)
             implementingMethods.Free();
         }
 
+        private void EmitMethod(IMethodDefinition methodDef, SymbolChange methodChange)
```

Comment:
>EmitMethod [](http://example.com/codeflow?start=21&length=10)

Not clear what "Emit" means here. The caller is called `CreateIndicesForNonTypeMembers` so I think `CreateIndicesForMethod` would be clearer